packages/@aws-cdk/aws-ecs-patterns/README.md

```diff
@@ -336,6 +460,22 @@ const scheduledFargateTask = new ecsPatterns.ScheduledFargateTask(this, 'Schedul
 });
 ```
 
+On Fargate Platform Version 1.4.0 or later, you may specify up to 200GiB of [ephemeral storage](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-task-storage.html#fargate-task-storage-pv14):
```

Comment:
Same as above: Can you clarify here that the minimum is 21 because 20 is provided by default?